src/sim_config.cpp

```diff
@@ -216,20 +216,30 @@ namespace Nos3
                 _config.put("nos3-configuration.common.log-config-file", "sim_log_config.xml");
             }
 
+            if (vm.count("help")) {
+                std::cout << generic << std::endl;
+                exit(1);
+            } else if (vm.count("version")) {
+                std::cout << "Version: " << VERSION << std::endl;
```

Comment:
Prints out blank when using shared folders. Should the version just be hard coded into a header file to avoid CMake magic?

![image](https://user-images.githubusercontent.com/6588622/229507683-1186b288-108f-4685-952d-740207874d26.png)

Reply:
Discussed with John and decided to not report version at all.